README.md

```diff
@@ -0,0 +1,81 @@
+![HTML](https://img.shields.io/badge/-HTML-orange) ![CSS](https://img.shields.io/badge/-CSS-blue) ![JavaScript](https://img.shields.io/badge/-JavaScript-yellow)
+
+# To-Do List App
+
+A sleek and intuitive To-Do List app built with JavaScript. This app helps you organize your day, manage your tasks efficiently, and boost your productivity. Whether it's marking off simple daily chores or tackling big projects, this app will keep you on track!. It uses local-storage to store the data.
+
+## 🌟 Features
+
+- Task Creation: Effortlessly add your tasks and keep track of everything you need to get done.
+
+- Task Completion: Mark tasks as completed with a simple checkbox and feel the satisfaction of checking them off.
+
+- Edit and Modify: Update or change tasks on the fly with an easy-to-use editing feature.
+
+- Task Deletion: Remove tasks that are no longer relevant with a quick delete button.
+
+- Smart Filters: Filter tasks by completion status (All, Completed, Incomplete) for a more organized experience.
+
+- Persistent Data: Tasks are saved locally using localStorage, ensuring your to-do list is always available when you return.
+
+- Responsive Design: Perfectly fits on both mobile and desktop screens, making it easy to manage your tasks on the go.
+
+### Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+Prerequisites
+VIsual Studio Code.
+A Web Browser (preferably Brave)
+
+### Setup
+
+Clone the GitHub Repository
+- git clone "https://github.com/RomeesaKamal/to-do-app.git"
+
+### Usage
+
+Start your server by clicking Go Live
+
+## Built With
+
+   **HTML/CSS & JS best practices:** Correct use of tags, elements, properties and syntax.
+
+   **GitHub flow:** Correct use of Branches for deployment and features development.
+
+## 🌎 Live Demo:
+- [Live Demo Link](https://RomeesaKamal.github.io/List-of-to-do-task/dist/)
+
+### 👤 **Romeesa Kamal**
+- GitHub: [@kamalromeesa](https://github.com/RomeesaKamal/)
+
```

Comment:
Please add Author heading, you miss it.
![image](https://github.com/user-attachments/assets/48e07835-a0bc-4670-b66d-f4e34fe83eb7)